jupyter/amd/pytorch/ubi9-python-3.9/Pipfile

```
@@ -0,0 +1,64 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[[source]]
+url = "https://download.pytorch.org/whl/nightly/rocm6.1/"
```

```suggestion
url = "https://download.pytorch.org/whl/rocm6.0/"
```

I don't like the `/nightly/` part. We should use stable releases, not nightly builds. So maybe like this?